src/camvidlog2/cli.py

```
@@ -52,64 +58,70 @@
+    queries: Annotated[list[str] | None, typer.Argument()] = None,
     outdir: Annotated[Path | None, typer.Option()] = None,
     db: Annotated[Path, typer.Option()] = Path("tmp.feather"),
     num: Annotated[int, typer.Option("--num", "-n", min=0)] = 10,
     roll: Annotated[int, typer.Option("--rolling", "-r", min=0)] = 0,
+    json: Annotated[Path | None, typer.Option()] = None,
 ):
-    df = data_load(db)
-    if df is None:
+    video_embeddings = data_load(db)
+    if video_embeddings is None:
         raise ValueError("Unable to load database")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)